samples/routing/data-distribution/Core_6/Client/CacheInvalidationHandler.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+#region CacheInvalidationHandler
+namespace DataDistribution
```

Comment:
It is used in the type filter so that the main endpoint does not load the data distribution types and vice versa.